docker-compose.yml

```diff
@@ -1,51 +1,50 @@
 version: "3.9"
 services:
-  dashboard:
-    image: louislam/uptime-kuma:1
```

Comment:
Why is it removed? Did we decide that we do not want to have it any longer?

Reply:
Deleted it cause it just blows up the compose-file and creates unwanted data. Was never used by anyone.